scripts/release/inline-resources.js

```diff
@@ -93,7 +93,9 @@ function inlineTemplate(content, urlResolver) {
     const templateContent = fs.readFileSync(templateFile, 'utf-8');
     const shortenedTemplate = templateContent
       .replace(/([\n\r]\s*)+/gm, ' ')
```

Comment:
That's a good point. Couldn't we just use gulp-clean-css to do proper minifying?